src/cls/lock/cls_lock_types.h

```diff
@@ -13,6 +13,9 @@
 #define LOCK_FLAG_MAY_RENEW 0x1    /* idempotent lock acquire */
 #define LOCK_FLAG_MUST_RENEW 0x2   /* lock must already be acquired */
 
+/// name of the lock used on objects to ensure layout stability during IO
+const std::string RADOS_LOCK_NAME("striper.lock");
```

Comment:
i'd suggest define it as a string_view like
```c++
inline constexpr std::string_view RADOS_LOCK_NAME{"striper.lock"};
```
smaller memory print than that of a full-blown `std::string`.